pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -1033,8 +1033,8 @@ private HandlerContext getHandlerContext(@Nullable TableConfig offlineTableConfi
       }
     }
 
-    // Disable Groovy if either offline or realtime table config disables Groovy
-    boolean disableGroovy = offlineTableDisableGroovyQuery | realtimeTableDisableGroovyQuery;
+    // Disable Groovy if both offline and realtime table config disable Groovy
```

Comment:
Why do we change this? We can only allow groovy when both offline and realtime table has it enabled

Reply:
I will improve this logic in that case. I was trying to handle `offline/ realtime only case`.

We need an extra `null` check for the table configs.

1. If both offline/realtime exists, allow if both got enabled.
2. if only one exists, allow if the specific table got enabled.

Reply:
@Jackie-Jiang pls review the updated code.